tox.ini

```diff
@@ -73,27 +39,54 @@ commands_post =
     coverage html
     coverage xml
 
-[testenv:docs]
-usedevelop = true
-extras = docs
-commands = sphinx-build {posargs:-E} -W -b html docs/src docs/build/html
+[testenv:lint]
+# this environement lints the Python code with flake8 (code linter), black (code
+# formatter), and isort (sorting of imports)
+skip_install = true
+deps =
+    flake8
+    flake8-bugbear
+    black
+    isort
+commands =
+    flake8 {[tox]lint_folders}
+    black --check --diff {[tox]lint_folders}
+    isort --check-only --diff {[tox]lint_folders}
 
 [testenv:format]
 # Abuse tox to do actual formatting. Users can call `tox -e format` to run
 # formatting on all files
+skip_install = true
 deps =
     black
     isort
 commands =
     black {[tox]lint_folders}
     isort {[tox]lint_folders}
 
+[testenv:docs]
+usedevelop = true
+# this environement builds the documentation with sphinx
+deps =
+    -r docs/requirements.txt
+commands = sphinx-build {posargs:-E} -W -b html docs/src docs/build/html
+
+[testenv:build]
```

Comment:
this test is not as useful as in equistore/rascaline since equisolve is (for now) a pure Python package. Does not hurt having it, but I don't expect this to catch many errors

Reply:
At least it is testing the release process via twine and checking the manifest file.